### PR TITLE
Oufitter :  "In cargo: x | stock: y"

### PR DESCRIPTION
**Pull request by whismerhill**
Just a small change for the way information is displayed.

Hope I didn't make any mistake.

EDIT: screenshot : http://steamcommunity.com/id/whispershill/screenshot/87096384653669617

PS: I don't know how to remove the two useless commits I did by mistake (de895be & 5b94c4c)

### CODE REVIEW

**Comment by Amazinite** [comment]
#1414

**Comment by whismerhill** [comment]
very well then.
let's argument my case
there's plenty of times where I like to keep whatever outfits I have in cargo
however when selling & buying over multiple different ships, I might lose track of the exact counts of each item. This allows to see exactly which items the player left in stock, and if he wants to he can then unselect ships and buy them back  to put them back in cargo.

As a matter of fact I have already lost plenty of outfits because things in stock are not shown if you also have some in cargo.

File: source/OutfitterPanel.cpp [hunk 1]
```diff
@@ -151,7 +151,15 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point, int scroll
 	}
 	if(player.Cargo().Get(outfit))
 	{
-		string count = "in cargo: " + to_string(player.Cargo().Get(outfit));
+		string count;
+		if(!outfitter.Has(outfit) and player.Stock(outfit) > 0)
```
Comment (Amazinite) [inline source/OutfitterPanel.cpp]: Would you want to use && instead of "and"? I don't see "and" being used like this anywhere else.

**Comment by sbeden** [comment]
Indeed, still, using cheat engine (specially with 3>2 is directly found), it is possible to get back to float when that happen.

**Comment by whismerhill** [comment]
@sbeden 
excuse me ? what the hell are you saying ? this seems like you made a mistake

**Comment by sbeden** [comment]
Not sure what happened,  this message was a reply for someone post about the current non display of both on board and local stock and the limit it create.

So it is fine to delete both of theses messages. 

Regard.

**Comment by whismerhill** [comment]
@sbeden 
click the cross icon next to each of your comments to delete them :)
regards.

**Pull request closed without merging**